Dockerfile

```diff
@@ -1,22 +1,22 @@
-FROM alpine:3.4
+FROM debian:stretch-slim
 
-ENV APP_PKGS="ca-certificates"
-ENV BUILD_PKGS="wget"
+RUN apt-get update -y && \
+    apt-get install --no-install-recommends -y \
+      ca-certificates wget && \
+    rm -rf /tmp/* /var/{cache,log,tmp} /var/lib/{apt,dpkg,cache,log}
 
-ENV OAUTH2_PROXY_VERSION="2.1"
-ENV OAUTH2_PROXY_PKG="oauth2_proxy-${OAUTH2_PROXY_VERSION}.linux-amd64.go1.6" \
-    OAUTH2_PROXY_SHA="3061e5b04bd14eeb9ec0ad1c9b324ba8d99d50eaadc5f528cdf4d21043828298"
-RUN apk update && \
-    apk upgrade && \
-    apk add $APP_PKGS $BUILD_PKGS && \
-    mkdir -p /var/tmp/oauth2_proxy && \
+ARG OAUTH2_PROXY_VERSION="2.2"
+ARG OAUTH2_PROXY_GOLANG_VERSION="1.8.1"
+ARG OAUTH2_PROXY_PKG_VERSION="2.2.0"
+ARG OAUTH2_PROXY_PKG="oauth2_proxy-${OAUTH2_PROXY_PKG_VERSION}.linux-amd64.go${OAUTH2_PROXY_GOLANG_VERSION}"
+ARG OAUTH2_PROXY_SHA="1c16698ed0c85aa47aeb80e608f723835d9d1a8b98bd9ae36a514826b3acce56"
+
+RUN mkdir -p /var/tmp/oauth2_proxy && \
     cd /var/tmp/oauth2_proxy && \
     wget --progress=dot:mega https://github.com/bitly/oauth2_proxy/releases/download/v${OAUTH2_PROXY_VERSION}/${OAUTH2_PROXY_PKG}.tar.gz && \
     echo "${OAUTH2_PROXY_SHA} *${OAUTH2_PROXY_PKG}.tar.gz" | sha256sum -c - && \
     tar xvf ${OAUTH2_PROXY_PKG}.tar.gz && \
-    cp /var/tmp/oauth2_proxy/${OAUTH2_PROXY_PKG}/oauth2_proxy /bin/ && \
-    apk del $BUILD_PKGS && \
-    rm -rf /var/cache/apk/*
+    cp /var/tmp/oauth2_proxy/${OAUTH2_PROXY_PKG}/oauth2_proxy /bin/
```

Comment:
Do we not need to cleanup `/var/tmp` here?

Reply:
Will address.

Reply:
Addressed in https://github.com/Intellection/docker-oauth2-proxy/pull/11/commits/1dddf4293e63c6aaad1465af0675f820e9df015e and https://github.com/Intellection/docker-oauth2-proxy/pull/11/commits/0f71fa5a22cec01f3580eb1aacfd501f831d0985.